README.md

```diff
@@ -59,6 +59,17 @@ You can refer to the following articles on the basics of Git and Github and also
 - [JavaScript](https://github.com/HarshCasper/Rotten-Scripts/tree/master/JavaScript)
 - [GoLang](https://github.com/HarshCasper/Rotten-Scripts/tree/master/Go)
 
+#### Tired of editing GitHub Profile README with new features?
+
+This tool provides an easy way to create a GitHub profile readme with the latest add-ons such as `visitors count`, `github stats`, etc.
+
+## 🚀 Demo 
+<a href="https://rahuldkjain.github.io/gh-profile-readme-generator" target="blank">
+<img src="https://img.shields.io/website?url=https%3A%2F%2Frahuldkjain.github.io%2Fgh-profile-readme-generator&logo=github&style=flat-square" />
+</a>
+
+Try the tool: [GitHub Profile README Generator](https://rahuldkjain.github.io/gh-profile-readme-generator)
```

Comment:
We don't need this. 

Maybe you had a very wrong conception around the Issue. I was perhaps looking for a Script that can generate a Profile README for the User and then push it to Github. We don't want our main README to be an advert for the tool. 

I'm sorry but I would not be able to merge the PR with this right here!